image_cropping/backends/base.py

```diff
@@ -1,6 +1,6 @@
 import abc
 
-from django.utils.translation import ugettext as _
+from django.utils.translation import gettext as _
```

Comment:
See the compatibility comment above.

Reply:
Also gettext is available in 2.2:
https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.translation.gettext